tests/performance/MemoryTest.cpp

```diff
@@ -159,6 +159,9 @@ void MemoryTest::RandomlyCancel(olp::client::CancellationToken token) {
  * Valgrind, heaptrack, other tools are used to collect the output.
  */
 TEST_P(MemoryTest, ReadNPartitionsFromVersionedLayer) {
+  // Enable only errors to have a short output.
+  olp::logging::Log::setLevel(olp::logging::Level::Error);
```

Comment:
```suggestion
  olp::logging::Log::setLevel(olp::logging::Level::Warning);
```
Still to much logging?